db/migrate/20211207070352_add_columns_to_keywords.rb

```diff
@@ -0,0 +1,9 @@
+class AddColumnsToKeywords < ActiveRecord::Migration[6.1]
+  def change
+    add_column :keywords, :ads_top_count, :integer
+    add_column :keywords, :ads_page_count, :integer
+    add_column :keywords, :non_ads_count, :integer
+    add_column :keywords, :total_links_count, :integer
+    add_column :keywords, :html, :text
```

Comment:
There will be always an html response, whether it is an error or success response, so how about to adding null constraint for this column?

Reply:
Currently, I only save the HTML response when the status is 200, so there is no error HTML response. Should we need to save the error HTML response?

Reply:
Up to you. If you don't want to store error response, then it is fine not to add null constraint